src/runner/fixture-hook-controller.ts

```diff
@@ -59,30 +59,52 @@ export default class FixtureHookController {
         return !!item && item.runningFixtureBeforeHook;
     }
 
+    private async _runFixtureBeforeHook (item: FixtureState, fn: Function): Promise<boolean> {
+        const shouldRunBeforeHook = !item.started && fn;
+
+        if (!shouldRunBeforeHook)
+            return !item.fixtureBeforeHookErr;
```

Comment:
Can we have the `fixtureBeforeHookErr` property set to some value here? AFAIU this condition is for the second hook, but if we have an error in the first hook we should not go here since we have `&&` sign between two `_runFixtureBeforeHook` calls